source/draw/RasterYuv420Semiplanar.ooc

```diff
@@ -247,4 +247,12 @@ RasterYuv420Semiplanar: class extends RasterYuvSemiplanar {
 		fileWriter close()
 	}
 	createPaintEngine: override func -> PaintEngine { Yuv420PaintEngine new(this) }
+	kean_draw_rasterYuv420Semiplanar_new: static unmangled func (width, height, stride: Int, monochromeData: Void*, uvData: Void*) -> This {
```

Comment:
No problem, but yes, you could (you would have to create a new branch at your last one and rebase my commit on top of your new one, but this works too).